multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -97,55 +80,22 @@ func NewMemberClusterAnnounceReconciler(client client.Client, scheme *runtime.Sc
 // Reconcile implements cluster status management on the leader cluster
 func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	memberAnnounce := &multiclusterv1alpha1.MemberClusterAnnounce{}
-	err := r.Get(ctx, req.NamespacedName, memberAnnounce)
+	var err error
+	memberID := getIDFromName(req.Name)
+	err = r.Get(ctx, req.NamespacedName, memberAnnounce)
```

Comment:
Nit: you can keep the previous version - err := r.Get(ctx, req.NamespacedName, memberAnnounce)